app/src/lib/repository-matching.ts

```diff
@@ -26,7 +26,8 @@ function matchRemoteWithAccount(account: Account, remote: string): GitHubReposit
 
   const owner = parsedRemote.owner
   const name = parsedRemote.name
-  if (parsedRemote.hostname === host && owner && name) {
+
+  if (host && parsedRemote.hostname.toLowerCase() === host.toLowerCase() && owner && name) {
```

Comment:
This was a wild guess that I had early on when investigating that never turned out to be meaningful but I still think that comparing host names should be done without taking case into account